### PR TITLE
First API testing

### PR DESCRIPTION
**Pull request by Maosorio92**

### CODE REVIEW

File: test/MyFirstApiConsume.test.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
+const axios = require('axios');
+const { expect } = require('chai');
+const { StatusCodes } = require('http-status-codes');
+
+describe('First Api Tests', () => {
+});
+
+it('Consume GET Service', async () => {
```
Comment (danielgalvis98) [inline test/MyFirstApiConsume.test.js]: The idea of ITs is to have them inside the describe, it is a section. (Same for the other its)

File: test/MyFirstApiConsume.test.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    const response = await axios.get('https://httpbin.org/ip');
+  
+    expect(response.status).to.equal(StatusCodes.OK);
+    expect(response.data).to.have.property('origin');
+
```
Comment (danielgalvis98) [inline test/MyFirstApiConsume.test.js]: Avoid this empty lines at the end of functions or methods

File: test/MyFirstApiConsume.test.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+      name: 'John',
+      age: '31',
+      city: 'New York'
+    };
+  
+    const response = await axios.get('https://httpbin.org/get', { query });
+  
+    expect(response.status).to.equal(StatusCodes.OK);
+    expect(response.config.query).to.eql(query);
+
```
Comment (danielgalvis98) [inline test/MyFirstApiConsume.test.js]: Avoid empty line

File: test/MyFirstApiConsume.test.js [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+  it('Consume HEAD Service', async () => {
+    const response = await axios.head('https://httpbin.org/get');
+  
+    expect(response.status).to.equal(StatusCodes.OK);
+    expect(response.headers).to.not.equal(null);
+    console.log("Status is ",response.status);
+    
+    
+
```
Comment (danielgalvis98) [inline test/MyFirstApiConsume.test.js]: Empty lines

File: test/MyFirstApiConsume.test.js [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+  it('Consume POST Service', async () => {
+    const response = await axios.post('https://httpbin.org/post',newuser);
+  
+    expect(response.status).to.equal(StatusCodes.OK);
+    expect(response.data.json).to.have.property("name");
+    console.log("Status is ",response.status);
+    console.log("Name of new user is ",response.data.json.name, " , age is ", response.data.json.age, " and activity is ", response.data.json.activity);
+    
+
```
Comment (danielgalvis98) [inline test/MyFirstApiConsume.test.js]: lines

File: test/MyFirstApiConsume.test.js [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+  it('Consume PATCH Service', async () => {
+    const response = await axios.patch('https://httpbin.org/patch',newuser);
+  
+    expect(response.status).to.equal(StatusCodes.OK);
+    expect(response.data.json).to.have.property("age");
+    console.log("Status is ",response.status);
+    
+    
+
```
Comment (danielgalvis98) [inline test/MyFirstApiConsume.test.js]: lines

File: test/MyFirstApiConsume.test.js [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+  it('Consume PUT Service', async () => {
+    const response = await axios.put('https://httpbin.org/put',newuser);
+  
+    expect(response.status).to.equal(StatusCodes.OK);
+    expect(response.data.json).to.have.property("born_in");
+    console.log("Status is ",response.status);
+    
+    
+
```
Comment (danielgalvis98) [inline test/MyFirstApiConsume.test.js]: lines

File: test/MyFirstApiConsume.test.js [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+  });
+
+  
+  it('Consume DELETE Service', async () => {
+    const response = await axios.delete('https://httpbin.org/delete',newuser);
+  
+    expect(response.status).to.equal(StatusCodes.OK);
+    expect(response.data.json).to.length.to.eql(null);
+        
+
```
Comment (danielgalvis98) [inline test/MyFirstApiConsume.test.js]: lines

File: test/MyFirstApiConsume.test.js [hunk 9]
```diff
@@ -0,0 +1,89 @@
...
+  });
+
```
Comment (danielgalvis98) [inline test/MyFirstApiConsume.test.js]: This last line of the file should not have any spaces or any character. Last line should be line 89, but it should be empty, only being created by the line break, which represents the end of file

Comment (Maosorio92) [reply test/MyFirstApiConsume.test.js]: Done!

File: test/MyFirstApiConsume.test.js (hunk 8 above)
Comment (Maosorio92) [reply test/MyFirstApiConsume.test.js]: Done!

File: test/MyFirstApiConsume.test.js (hunk 7 above)
Comment (Maosorio92) [reply test/MyFirstApiConsume.test.js]: Done!

File: test/MyFirstApiConsume.test.js (hunk 6 above)
Comment (Maosorio92) [reply test/MyFirstApiConsume.test.js]: Done!

File: test/MyFirstApiConsume.test.js (hunk 5 above)
Comment (Maosorio92) [reply test/MyFirstApiConsume.test.js]: Done!

File: test/MyFirstApiConsume.test.js (hunk 4 above)
Comment (Maosorio92) [reply test/MyFirstApiConsume.test.js]: Done!

File: test/MyFirstApiConsume.test.js (hunk 3 above)
Comment (Maosorio92) [reply test/MyFirstApiConsume.test.js]: Done!

File: test/MyFirstApiConsume.test.js (hunk 2 above)
Comment (Maosorio92) [reply test/MyFirstApiConsume.test.js]: Done!

File: test/MyFirstApiConsume.test.js (hunk 1 above)
Comment (Maosorio92) [reply test/MyFirstApiConsume.test.js]: Done!

**Comment by manuelq12** [comment]
@Maosorio92 Hey man, just a quick question, why do you need the console logs?
The idea is that assertions show you the results you receive, so

**Comment by Maosorio92** [comment]
@manuelq12 Hello!
I can erase them, it is just to see that everything is going well, 
I will erase them now
Thank you!

**Comment by Maosorio92** [comment]
@manuelq12 
I have already erased console logs in Rev3.
Thank you! I wait for your approval to continue with next step.

**Review by manuelq12 — APPROVED** [review]
LGTM 🎖️

**Review by danielgalvis98 — APPROVED** [review]

**Pull request merged**